app/src/main/kotlin/com/wire/android/ui/home/conversationlist/common/RowItem.kt

```diff
@@ -10,12 +11,17 @@ import com.wire.android.ui.common.SurfaceBackgroundWrapper
 import com.wire.android.ui.theme.Dimensions
 
 
+//TODO: added onRowClick only for UI-Design purpose
 @Composable
-fun RowItem(content: @Composable RowScope.() -> Unit) {
+fun RowItem(onRowItemClick: () -> Unit, content: @Composable RowScope.() -> Unit) {
     SurfaceBackgroundWrapper(modifier = Modifier.padding(Dimensions.conversationItemPadding)) {
         Row(
             verticalAlignment = Alignment.CenterVertically,
-            modifier = Modifier.padding(Dimensions.conversationItemRowPadding)
+            modifier = Modifier
+                .padding(Dimensions.conversationItemRowPadding)
```

Comment:
I'm not sure these paddings are actually necessary, it narrows even more the clickable area, which should contain the whole height and width of each element
<img width="208" alt="Captura de pantalla 2022-01-31 a las 13 13 45" src="https://user-images.githubusercontent.com/2468164/151792100-ad127263-67df-4754-b501-636042cf579c.png">
.